share/wake/lib/core/print.wake

```diff
@@ -61,20 +61,20 @@ export def makeLogLevel2 (name: String) (colour: Option Colour) (intensity: Opti
     def set name code = prim "colour"
     def colourCode = match colour
         None   = 0
-        Some c = 8 + match c
-            Black   = 0
-            Red     = 1
-            Green   = 2
-            Yellow  = 3
-            Blue    = 4
-            Magenta = 5
-            Cyan    = 6
-            White   = 7
+        Some c = match c
+            Black   = 8
+            Red     = 9
+            Green   = 10
+            Yellow  = 11
+            Blue    = 12
+            Magenta = 13
+            Cyan    = 14
+            White   = 15
     def intensityCode = match intensity
         None   = 0
-        Some i = 16 * match i
-            Dim    = 1
-            Bright = 2
+        Some i = match i
+            Dim    = 16
+            Bright = 32
```

Comment:
What does the formatter turn this into currently?

Reply:
```
export def makeLogLevel2 (name: String) (colour: Option Colour) (intensity: Option Intensity): LogLevel =
    def set name code = prim "colour"

    def colourCode = match colour
        None -> 0
        Some c ->
            8
            + match c
                Black -> 0
                Red -> 1
                Green -> 2
                Yellow -> 3
                Blue -> 4
                Magenta -> 5
                Cyan -> 6
                White -> 7

    def intensityCode = match intensity
        None -> 0
        Some i ->
            16
            * match i
                Dim -> 1
                Bright -> 2

    def _ = set name (colourCode + intensityCode)

    LogLevel name
```

I  don't mind it, but you mentioned it as being something you didn't like back on the OG bikeshed  PR

Reply:
Well that makes me want to puke personally so I'm glad you changed it. I don't think this pattern is that common but we should probably fix this after MVP. I think `8 + match c` shouldn't require indent